enrichment-web/src/main/java/eu/europeana/enrichment/web/service/EnrichmentTranslationService.java

```diff
@@ -20,7 +18,7 @@
 	 * 									a ID will be returned
 	 * @throws							HttpException
 	 */
-	public String translate(EnrichmentTranslationRequest requestParam, boolean process) throws HttpException, Exception;  
+	public void translate(EnrichmentTranslationRequest requestParam, boolean process) throws HttpException, Exception;  
```

Comment:
for asynchronuous translations with eTranslation the id ofthe request is needed to fetch the translation

Reply:
Yes, that is done here: eu.europeana.enrichment.translation.service.impl.ETranslationEuropaServiceImpl.translateText, and the code above should be ok.